docs/release-notes/latest-changes.md

```diff
@@ -28,6 +28,56 @@ All Rights Reserved. (Contributor contact(s):________________[Insert hyperlink/a
 
 Date       | Change
 ---------  | -------------
+2022-02-03 | fix terminator history
+2022-02-04 | barman: update to 2.18
+2022-02-04 | Firefox Version 91.5.1
+2022-02-03 | sbcl: update to 2.2.1
+2022-02-03 | harfbuzz: update to 3.3.1
+2022-02-03 | chezmoi: update to 2.11.1
+2022-02-02 | screen: update to 4.9.0
+2022-02-02 | findutils: update to 4.9.0
+2022-02-02 | Upgrade of terminator to 2.1.1 with python 3.9
+2022-02-02 | graphicsmagick: update to 1.3.37
+2022-02-02 | sendmail/libmilter - updated to latest version 8.17.1
+2022-02-01 | fonts/terminus: update to 4.49.1
+2022-02-01 | bcc: update to 0.16.21
+2022-02-01 | opendbx - Native DBExpress driver v1.4.6 (mysql, postgres, odbc, sqli…
+2022-02-01 | update nodejs-14 to 14.19.0
+2022-02-01 | Make GNU-M4 64bit only
+2022-02-01 | drakma: update to 2.0.8
+2022-02-01 | cl-ppcre: update to 2.1.1
+2022-02-01 | chunga: update to 1.1.7
+2022-02-01 | pigz: update to 2.7
+2022-02-01 | disable some failing packages until we can provide fixes
+2022-02-01 | puri: update to 1.5.7
```

Comment:
The entry ""disable some failing packages..." should also be removed.

Reply:
Done